subsys/bluetooth/mesh/cdb.c

```diff
@@ -64,10 +64,11 @@ struct app_key_val {
 	uint8_t  val[2][16];
 } __packed;
 
-/* IV Index & IV Update information for persistent storage. */
+/* Network information for persistent storage. */
 struct net_val {
 	uint32_t iv_index;
 	bool  iv_update;
+	uint16_t lowest_avail_addr;
```

Comment:
How are you dealing with stored values written to flash using prior versions with the smaller struct size? It looks like the cdb_net_set() function would simply fail to restore anything in that case. Is that what we want and the best we can do?

Reply:
They will be dropped. CDB is still marked as experimental, that is why I though changing the struct is OK.

Reply:
Updated to try recovering previous struct. Please have a look.